packages/kit/src/runtime/client/ambient.d.ts

```diff
@@ -7,11 +7,12 @@ declare module '__GENERATED__/client-manifest.js' {
 	export const nodes: CSRPageNodeLoader[];
 
 	/**
-	 * A map of `[routeId: string]: [errors, layouts, page]` tuples, which
+	 * A map of `[routeId: string]: [leaf, layouts, errors]` tuples, which
 	 * is parsed into an array of routes on startup. The numbers refer to the
-	 * indices in `nodes`.
+	 * indices in `nodes`. The route layout and error nodes are not referenced,
+	 * they are always number 0 and 1 and always apply.
 	 */
-	export const dictionary: Record<string, [number[], number[], number]>;
+	export const dictionary: Record<string, [number, number[]?, number[]?]>;
```

Comment:
For clarity, I think we should use named tuples here

```suggestion
	export const dictionary: Record<string, [leaf: number, layouts?: number[], errors?: number[]]>;
```